roles/openshift_storage_ceph/tasks/ceph_deploy.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: create minimal ceph cluster
+  oc_obj:
+    namespace: ""
+    kind: pod
+    name: rook-ceph
+    state: present
+    files:
+      - "{{ tmpdir }}/cluster.yml"
+
+- name: wait for the initial ceph-mon to be ready
+  oc_obj:
+    namespace: rook-ceph
```

Comment:
This won't work on default OVS plugin - for some reason all rook components should be placed in `rook-ceph-system` otherwise two namespaces won't be able to communicate

Reply:
What's OVS? The current code has been tested many times and works.
`rook-ceph-system` is for the operator and agents where `rook-ceph` is for the Ceph cluster.

Reply:
OpenVSwitch, see https://github.com/rook/rook/issues/2045